### DISCUSSION TITLE
How to implement "European User Consent" in react-native app and combine it with inrestitialAd, if necessary?

### DISCUSSION THREAD

**Discussion by Deodes** [invertase/react-native-firebase#5331] [Q&A]
Trying to provide functionality for asking users of what kind of ads they would like to see. But actually, I don't understand in what order you need to send requests and most importantly where (I've got App.js and other screen, where ads have to be shown). I would be very grateful for any example

**Comment by mikehardy** [comment]
Check the pinned issue at the top of the issues list to get access to the documentation, sorry it's not easily available at the moment: https://github.com/invertase/react-native-firebase/issues/5329
